Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -91,14 +107,16 @@ const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
   };
 
   /**
-   * Search items with specified search criteria.
+   * Search items with specified search criteria and show a spinner when the search is in progress.
    */
   const search = (): void => {
+    setShowSpinner(true);
     searchItems(searchOptions)
       .then((items: OEQ.Common.PagedResult<OEQ.Search.SearchResultItem>) =>
         setPagedSearchResult(items)
       )
-      .catch((error: Error) => handleError(error));
+      .catch((error: Error) => handleError(error))
```

Comment:
Since handleError's signature is also a single param with error, this can likely be folded.

```suggestion
      .catch(handleError)
```